### ISSUE TITLE
Question: Does training with  pretrained model weights require access to lables? 

### ISSUE THREAD

**Issue by minertom** [AlexeyAB/darknet#6399]
I am making an assumption that if I use a pre-trained model, such as yolov3-openimages.weights or darknet53.conv.74 and train on images already in a specific dataset, then I would not need to supply labels for the images that I wish to train on. 

My understanding is that I would only need to create labels for images in a custom dataset. Is that correct?

Thank You
Tom

**Comment by AlexeyAB** [comment]
No. 
You need labels for all images.

**Comment by minertom** [comment]
Kindly,

Where to put label/annotations directory? Which file points to the label/annotations directory? I am confused about the paths.

Thank You
Tom

**Comment by AlexeyAB** [comment]
For example, you use MS COCO pre-trained weights that is trained for these classes: https://github.com/AlexeyAB/darknet/blob/master/data/coco.names

* if you want to train for the same classes, then you should use MSCOCO dataset + your dataset (all images and txt files in 1 directory)

* if you want to train for different classes, then you should use only your dataset (all images and txt files in 1 directory)

no matter what pre-prepared weights you are using.

**Comment by minertom** [comment]
Thank You, I might be fighting a language understanding issue, so I apologize.

If I understand you correctly, for example, MS COCO pre trained weights are trained for classes in [https://github.com/AlexeyAB/darknet/blob/master/data/coco.names](url), as you say.

I believe that what you are saying is that "if you want to train for the same classes" means that if you want to train **additional images** of those same classes then use MSCOCO dataset (all jpg images  already in the MSCOCO dataset) PLUS the new images that you wish to add and their corresponding text files. In such a case, I believe that you are saying that you need a new set of annotations which contains annotations for the original set of images in the MSCOCO dataset PLUS the new annotations for the new added images. One txt file containing all annotations.

Is the above correct?

Then it would follow that for different classes, or classes not in the original MSCOCO dataset, use only your new images and the txt files with the annotations of those new images.

I appreciate your patience.

Would the same hold if you are only attempting to retrain on a smalll subset of the classes? So, for example, a class that is already in the dataset but only one class? Is retraining of the pretrained weights going to gain us anything?

Thank You
Tom
